lib/gitarro/backend.rb

```diff
@@ -174,6 +164,20 @@ def magicword(pr_number, context)
 
   private
 
+  # Show a message stating if there are opened PRs or not
+  def print_pr_resume(prs)
+    unless prs.empty?
```

Comment:
maybe as nitpick we should use 
``` if prs.any?``` 
this is more readable imho

Reply:
Hehehe, I was thing the same, but as you suggested using unless.. :-D

Reply:
Fixed. Running tests and pushing in a few minutes.